api/plugin_support.h

```diff
@@ -133,17 +137,78 @@ struct stack_frame {
 };
 
 /* Public functions */
+
+/**
+ * @brief Registers a new mambo plugin.
+ * 
+ * Registers a new mambo plugin and allocates a new mambo context A mambo
```

Comment:
Mix of "MAMBO" and "mambo". Please make it uniform. And missing dot after context.